OpenSim/Sandbox/CMakeLists.txt

```diff
@@ -18,6 +20,10 @@ endforeach()
 foreach(exec_file ${TO_COMPILE})
     get_filename_component(_target_name ${exec_file} NAME_WE)
     add_executable(${_target_name} EXCLUDE_FROM_ALL ${exec_file})
+	if(${exec_file} MATCHES "futureTaskSpace.cpp")
+		add_library(taskSpaceLib STATIC ${TASK_SPACE_FILES})
+		target_link_libraries(${_target_name} taskSpaceLib)
+	endif()
```

Comment:
It might be cleaner to move `futureTaskSpace.cpp` to the `TaskSpace` folder and build it there?